repositories/PageRepository.js

```diff
@@ -68,33 +62,42 @@ module.exports.insert = function insert(args) {
       )
       .then(fromDb);
   });
-};
+}
 
-module.exports.update = function update(args) {
+function update(args) {
   const repository = getRepositoryForType(args);
   return repository.update(args);
-};
+}
 
-module.exports.remove = function remove(id) {
+function remove(id) {
   return Page.update(id, { isDeleted: true })
     .then(head)
     .then(fromDb);
-};
+}
 
-module.exports.undelete = function(id) {
+function undelete(id) {
   return Page.update(id, { isDeleted: false })
     .then(head)
     .then(fromDb);
-};
+}
 
-module.exports.move = ({ id, sectionId, position }) => {
+function move({ id, sectionId, position }) {
   return db.transaction(trx =>
     movePage(trx, { id, sectionId, position }).then(fromDb)
   );
-};
+}
 
-module.exports.getPosition = ({ id }) => {
-  return Page.findById(id).then(({ SectionId }) =>
-    getPosition(db, SectionId, id)
-  );
-};
+function getPosition({ id }) {
+  return getById(id).then(get("position"));
+}
+
+Object.assign(module.exports, {
+  findAll,
+  get: getById,
+  insert,
+  update,
+  remove,
+  undelete,
+  move,
+  getPosition
+});
```

Comment:
Nice tidy up 👍

Reply:
It's a clever trick I saw here: https://twitter.com/getify/status/991427888359596032.

It saves on repetitious `module.export.whatever`, and prevents circular dependencies from causing issues.